src/main/java/com/heeverse/ticket/service/TicketService.java

```diff
@@ -60,14 +59,15 @@ private List<GradeTicket> saveGradeTicket(TicketRequestDto ticketRequestDto) {
     private void saveTicket(List<GradeTicket> gradeTickets, TicketRequestDto ticketRequestDto) {
 
         List<Ticket> tickets = gradeTickets.stream()
-                .flatMap(grade ->
-                    IntStream.range(0, grade.getTicketCount())
-                            .mapToObj(idx -> new Ticket(new TicketSerialNumber(
-                                        new TicketSerialTokenDto(ticketRequestDto.concertDate(), ticketRequestDto.concertSeq(), grade, idx)
-                                    ), grade)
-                            )
-                )
-                .toList();
+            .flatMap(grade ->
+                IntStream.range(0, grade.getTicketCount())
+                    .mapToObj(idx -> new Ticket(new TicketSerialNumber(
+                            new TicketSerialTokenDto(ticketRequestDto.concertDate(),
+                                ticketRequestDto.concertSeq(), grade, idx)
+                        ), grade)
+                    )
+            )
+            .toList();
```

Comment:
stream의 method depth가 너무 깊어서 어떤 코드인지 한눈에 들어오지 않습니다~
이러면 이후에 이 코드를 봤을 때 이게 어떤것을 의도하는  코드인지 확인하기가 어렵습니다.

이런 경우, 여러 라인으로 쪼개지는 로직은 모두 메서드로 분리해주시는게 좋습니다.
최대한 읽기 편안하게 만들어야 직접 만든 로직을 원활하게 읽고 파악할 수 있습니다
```suggestion
gradeTickets.stream()
            .flatMap(grade -> createTickets(grade, ticketRequestDto).stream())
            .toList();
...  

private List<Ticket> createTickets(~~~) {
          
```